chaos.py

```diff
@@ -19,6 +19,11 @@
 import patch
 
 
+if not os.path.isfile("usestartscript"):
+    open("usestartupscript", 'a').close()
```

Comment:
Nice catch, but this probably isn't getting approved so I won't bother to fix that.